proximal/examples/test_deconv_sv_psf.py

```diff
@@ -0,0 +1,58 @@
+# Spatially Varying PSF Deconvolution Algorithm based on:
+# Flicker & Rigaut, 2005 https://doi.org/10.1364/JOSAA.22.000504
+
+import sys
+
+sys.path.append('../../')
```

Comment:
> Thanks for fixing it. Not sure, what was the issue that reshape fixed.

Great! I believe that was the `sys.path.append('../../'` command that resolved your issue. We have not released `ProxImaL` to `pip` for quite some time. Perhaps we should do that, right after merging your hot fix in `comp_graph.py`.